### PR TITLE
Create CSS in JavaScript

### PR DESCRIPTION
**Pull request by nathpaiva**
To remove dependency of CSS import when use the library, I created a method that add a CSS from JavaScript.

The method use breaking point of gaiden and now the users can customize with own values

### CODE REVIEW

**Comment by iondrimba** [comment]
@nathpaiva I would also update the README part where it says `After install, you will have to add the stylesheet (dist/media-query.css) to your page via link tag or import it to your existing style manifest.` 

Can you verify if we still have 💚 Maintainability and 💚 coverage  😄

**Review by iondrimba — DISMISSED** [review]

File: spec/index.spec.js [hunk 1]
```diff
@@ -3,26 +3,60 @@ import MediaQueryContent from '../src/dhalsim';
...
+  });
+
+  describe('renderCSS', () => {
+    beforeEach(() => {
+      document.body.innerHTML = __html__['spec/fixtures/renderCss.html'];
+    });
+
+    it('call renderCSS', () => {
+      mediaQueryContent = new MediaQueryContent().init();
+      const element = document.querySelector('style');
```
Comment (eduardojmatos) [inline spec/index.spec.js]: `const` before anything, when it is possible =D

File: src/dhalsim.js [hunk 2]
```diff
@@ -1,16 +1,61 @@
...
 };
 
 export default class Dhalsim {
   constructor(options) {
     const defaultOptions = Object.assign({}, defaultDhalsimOptions, options);
     this.breakingPointValue = defaultOptions.breakingPointValue;
 
+    this.tablet = defaultOptions.tablet;
+    this.desktop = defaultOptions.desktop;
+    this.wide = defaultOptions.wide;
```
Comment (eduardojmatos) [inline src/dhalsim.js]: This assignments are not necessary, because `Object.assign` already assign this properties, with `defaultDhalsimOptions`.

**Review by eduardojmatos — CHANGES_REQUESTED** [review]
Just a minor fix.

This PR is very <3, thank you!!! \o/

File: src/dhalsim.js (hunk 2 above)
Comment (eduardojmatos) [reply src/dhalsim.js]: Sorry, `Object.assign`, line 10, are not assigning `this`. We could make some thing directly like this: 

```JavaScript
Object.assign(this, defaultDhalsimOptions, options);
```

And delete const `defaultOptions` =D

**Review by iondrimba — APPROVED** [review]

**Review by williamcosta — APPROVED** [review]
Nice 🚀

**Review by felipeorlando — APPROVED** [review]

**Review by eduardojmatos — APPROVED** [review]

**Review by iondrimba — APPROVED** [review]

**Pull request merged**